src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml

```diff
@@ -1,19 +1,32 @@
 ﻿<UserControl xmlns="https://github.com/avaloniaui"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:conv="using:Avalonia.Diagnostics.Converters"
-             xmlns:local="using:Avalonia.Diagnostics.Views"
-             xmlns:controls="using:Avalonia.Diagnostics.Controls"
-             xmlns:vm="using:Avalonia.Diagnostics.ViewModels"
-             xmlns:lb="using:Avalonia.Diagnostics.Behaviors"
+             xmlns:conv="clr-namespace:Avalonia.Diagnostics.Converters"
```

Comment:
Why clr-namespace?

Reply:
Only for this issue #12974. I make PR https://github.com/kekekeks/XamlX/pull/94 , has not yet been approved.

Reply:
pinged about this PR